syntax = "proto3";

package hermes.example;

option java_package = "com.hermes.example";
option java_multiple_files = true;

service HelloWorldService {

    rpc SayHello(SayHelloRequest) returns (SayHelloResponse) {}
}

message SayHelloRequest {

}

message SayHelloResponse {
    string name = 1;
}
